app/config/frontendAppConfig.scala

```diff
@@ -14,9 +14,9 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.softdrinksindustrylevy.config
+package config
```

Comment:
could we please use sdil.config instead? And so on and so forth.

Reply:
Done